system/atomic_image.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION='''
+---
+module: atomic_image
+short_description: Manage the container images on the atomic host platform
+description:
+    - Manage the container images on the atomic host platform
+    - Allows to execute the commands on the container images
+    - Commands ```install``` and ```run``` will download the container image if not present already
+version_added: "2.2"
+author: "Saravanan KR @krsacme"
+notes:
+    - Host should be an atomic platform (verified by existence of '/run/ostree-booted' file)
+requirements:
+  - atomic
+options:
+    name:
+        description:
+          - Name of the container image
+        required: True
+        default: null
+    upgrade:
+        description:
+          - Forcefully upgrade if the container, even if the container is already running
+        required: False
+        choices: ["yes", "no"]
+        default: no
+    state:
+        description:
+          - The state of the container image
+        required: False
+        choices: ["started", "stopped"]
```

Comment:
@detiber I have named according to the previous comments from @tima which is 
_Look at other similar modules (don't look at the Docker module -- that one is an anti-pattern we are going to address soon enough) and you'll see state use values like "present", "latest", "absent" if its for deployment or package management. Use "started", "restarted" and "stopped" for process management. I'd hesitate about having a module that does both deployment and service management._